stories/setup/webpack.config.babel.js

```diff
@@ -1,58 +1,18 @@
-import 'core/polyfill';
+import { getPlugins, getRules } from '../../webpack-common';
+import webpackConfig, {
+  babelOptions as defaultBabelOptions,
+} from '../../webpack.dev.config.babel';
 
-import path from 'path';
-
-import config from 'config';
-import autoprefixer from 'autoprefixer';
-import webpack from 'webpack';
-
-import { getClientConfig } from 'core/utils';
+// We need to add the docgen plugin to get Flow to work.
+const babelOptions = {
+  ...defaultBabelOptions,
+  plugins: [...defaultBabelOptions.plugins, 'react-docgen'],
+};
 
 module.exports = {
+  ...webpackConfig,
   module: {
-    rules: [
-      {
-        test: /\.svg$/,
-        use: [{ loader: 'svg-url-loader', options: { limit: 10000 } }],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader', options: { importLoaders: 2 } },
-          {
-            loader: 'postcss-loader',
-            options: {
-              plugins: [
-                autoprefixer({
-                  browsers: ['last 2 versions'],
-                  grid: false,
-                }),
-              ],
-            },
-          },
-          { loader: 'sass-loader', options: { outputStyle: 'expanded' } },
-        ],
-      },
-    ],
-  },
-  // The following plugins are needed to help handle
-  // server-side imported components such as core/logger.
-  // Is this the best way??
-  plugins: [
-    new webpack.DefinePlugin({
-      CLIENT_CONFIG: JSON.stringify(getClientConfig(config)),
-    }),
-
-    new webpack.NormalModuleReplacementPlugin(
-      /config$/,
-      'core/client/config.js',
-    ),
-  ],
-  resolve: {
-    alias: {
-      normalize: 'normalize.css/normalize.css',
-    },
-    modules: [path.resolve('./src'), 'node_modules'],
+    rules: [...getRules({ babelOptions, bundleStylesWithJs: true })],
   },
+  plugins: [...getPlugins()],
```

Comment:
Is this needed?

Reply:
hmm..I thought I saw an issue so I thought I had to override dev but actually it's fine so I've removed this.